cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -167,15 +169,14 @@ private CruiseControl(Reconciliation reconciliation, HasMetadata resource, Share
      * Creates an instance of the Cruise Control model from the custom resource. When Cruise Control is not enabled,
      * this will return null.
      *
-     * @param reconciliation                Reconciliation marker used for logging
-     * @param kafkaCr                       The Kafka custom resource
-     * @param versions                      Supported Kafka versions
-     * @param kafkaBrokerNodes              List of the broker nodes which are part of the Kafka cluster
-     * @param kafkaStorage                  A map with storage configuration used by the Kafka cluster and its node pools
-     * @param kafkaBrokerResources          A map with resource configuration used by the Kafka cluster and its broker pools
-     * @param sharedEnvironmentProvider     Shared environment provider
-     *
-     * @return  Instance of the Cruise Control model
+     * @param reconciliation            Reconciliation marker used for logging
+     * @param kafkaCr                   The Kafka custom resource
+     * @param versions                  Supported Kafka versions
+     * @param kafkaBrokerNodes          List of the nodes which are part of the Kafka cluster
+     * @param kafkaStorage              A map with storage configuration used by the Kafka cluster and its node pools
+     * @param kafkaBrokerResources      A map with resource configuration used by the Kafka cluster and its node pools
+     * @param sharedEnvironmentProvider A Interface to be implemented for returning an instance of the environment variables shared by all containers.
+     * @return Instance of the Cruise Control model
```

Comment:
Careful on IDE indentation, the spacing fixes are fine between the parameters, just watch the new line removal between the parameters and the return line